cryptography/src/androidMain/kotlin/com/wire/kalium/cryptography/ProteusClientImpl.kt

```diff
@@ -87,10 +87,10 @@ actual class ProteusClientImpl actual constructor(rootDir: String, userId: Strin
 
     companion object {
         private fun toPreKey(preKey: PreKey): com.wire.cryptobox.PreKey =
-            com.wire.cryptobox.PreKey(preKey.id, Base64.decode(preKey.encodedData, Base64.DEFAULT))
+            com.wire.cryptobox.PreKey(preKey.id, Base64.decode(preKey.encodedData, Base64.NO_WRAP))
```

Comment:
Is there a way to rename sensibly these 2 different implementations of `PreKey` so that we can distinguish which one comes from Cryptobox and which one from the ProteusClientImpl ?

Reply:
Yeaaah.

Maybe we can create an typealias inside `cryptography` module to refeer to the native stuff. Something like `NativePreKey` or something.